protobuf/plugin/deepcopy/test/deepcopy.pb.go

```diff
@@ -46,93 +46,93 @@ var _ = math.Inf
 const _ = proto.GoGoProtoPackageIsVersion2 // please upgrade the proto package
 
 type BasicScalar struct {
-	Field1  float64 `protobuf:"fixed64,1,opt,name=Field1,json=field1,proto3" json:"Field1,omitempty"`
```

Comment:
These changes look unrelated. Perhaps you are using a different version of protoc. CI is using https://github.com/google/protobuf/releases/download/v3.0.2/protoc-3.0.2-linux-x86_64.zip